activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -214,6 +214,7 @@ def initialize(types, name, temporary, options, as = nil)
         @name = name
       end
 
+      # Returns an array of ColumnDefinitions for the columns of the table.
```

Comment:
This is a tricky one. If we document that this method returns `ColumnDefinition` objects a user expects to find documentation on `ColumnDefinition` to explore possible interactions. Currently that class and it's subclasses are `:nodoc:`. /cc @rafaelfranca @carlosantoniodasilva